src/components/AppSidebar/AppSidebar.vue

```diff
@@ -336,9 +402,13 @@ export default {
 		 * @param {Event} event submit event
 		 */
 		onSubmitTitle(event) {
+			// Disable editing
+			this.$emit('update:titleEditable', false)
 			this.$emit('submit-title', event)
 		},
 		onDismissEditing() {
+			// Disable editing
+			this.$emit('update:titleEditable', false)
 			this.$emit('dismiss-editing')
 		},
```

Comment:
```suggestion
		onSubmitTitle(event) {
			// Disable editing
			this.$emit('update:titleEditable', false)
		},
		onDismissEditing() {
			// Disable editing
			this.$emit('update:titleEditable', false)
		},
```
I'm not sure this is something we want anymore?
We can already listen to the `update:titleEditable` event now.

Reply:
Same for the `this.$emit('input-title', event)`, doesn't make sense to have both, no? :)

Reply:
> I'm not sure this is something we want anymore?
> We can already listen to the `update:titleEditable` event now.

I think it would be good to emit `submit-title` and `dismiss-editing` separately, since you can then distinguish between the user pressing the `ESC` button and really submitting the title change by the button or clicking outside the input. When pressing ESC, as a user I wouldn't expect the title to be changed, but reset to the initial value. This wouldn't be possible if we only emit `update:titleEditable`.

Reply:
As for the `input-title` emission, I think we could remove it (although I didn't test this yet).

Reply:
> 
> 
> As for the `input-title` emission, I think we could remove it (although I didn't test this yet).

I will test this in the evening and remove it if possible.

Reply:
I think it's fine to remove the `input-title` event. Although it is a breaking change, I didn't find any repository in the nextcloud org which uses this. So I guess no one will notice the event is gone.